Modules/BiddingManager/GUI.lua

```diff
@@ -51,11 +52,27 @@ end
 local BiddingManagerGUI = {}
 function BiddingManagerGUI:Initialize()
     LOG:Trace("BiddingManagerGUI:Initialize()")
+    self:InitializeDB()
+    self:RegisterPlayerLogoutEvent()
     self:Create()
     self:RegisterSlash()
     self._initialized = true
 end
 
+function BiddingManagerGUI:InitializeDB()
+    local db = MODULES.Database:GUI()
+    if not db.bidding then
+        db.bidding = { }
+    end
+    self.db = db.bidding
+end
+
+function BiddingManagerGUI:RegisterPlayerLogoutEvent()
```

Comment:
This should be a local function; probably it shouldn't be a function at all..

Just register it in `Initialize`:
```lua
EventManager:RegisterEvent({"PLAYER_LOGOUT"}, function() StoreLocation(self) end)
```